sdk/textanalytics/Azure.AI.TextAnalytics/src/Generated/Models/AnalyzeTasks.Serialization.cs

```diff
@@ -24,7 +24,7 @@ internal static AnalyzeTasks DeserializeAnalyzeTasks(JsonElement element)
             Optional<IReadOnlyList<KeyPhraseExtractionTasksItem>> keyPhraseExtractionTasks = default;
             Optional<IReadOnlyList<EntityLinkingTasksItem>> entityLinkingTasks = default;
             Optional<IReadOnlyList<SentimentAnalysisTasksItem>> sentimentAnalysisTasks = default;
-            Optional<IReadOnlyList<TasksStateTasksExtractiveSummarizationTasksItem>> extractiveSummarizationTasks = default;
+            Optional<IReadOnlyList<ExtractiveSummarizationTasksItem>> extractiveSummarizationTasks = default;
```

Comment:
Renamed `TasksStateTasksExtractiveSummarizationTasksItem` to `ExtractiveSummarizationTasksItem` internally. It makes no difference in the public API, but other `<...>TaskItem` types were renamed as well, so I'm just following the pattern to keep them organized.

Reply:
just to easy the use internally. it was getting hard to read those crazy types :P